docs/api/sql/Function.md

```diff
@@ -1234,7 +1234,8 @@ MULTIPOLYGON (((-2 -3, -3 -3, -3 3, -2 3, -2 -3)), ((3 -3, 3 3, 4 3, 4 -3, 3 -3)
 
 Introduction:
 
-Transform the Spatial Reference System / Coordinate Reference System of A, from SourceCRS to TargetCRS
+Transform the Spatial Reference System / Coordinate Reference System of A, from SourceCRS to TargetCRS.
+For SourceCRS and TargetCRS, WKT format is also available.
```

Comment:
Please add "since `v1.3.1`". Please also add the same sentence to the Flink SQL doc. Thank you!

Reply:
I add the comments.